src/Radicle/Internal/Pretty.hs

```diff
@@ -164,6 +164,15 @@ renderCompactPretty = renderStrict . layoutCompact . prettyV
 renderPretty :: PrettyV v => PageWidth -> v -> Text
 renderPretty pg = renderStrict . layoutSmart (LayoutOptions pg) . prettyV
 
+-- | 'renderPretty', but with unbounded width.
+--
+-- Example:
+--
+-- >>> renderPrettyUnbounded (asValue (List [String "hi", String "there"]))
+-- "(\"hi\" \"there\")"
+renderPrettyUnbounded :: PrettyV v => v -> Text
+renderPrettyUnbounded = renderPretty Unbounded
```

Comment:
Can you add a test somewhere that generates arbitrary values that don't include strings with whitespace, and then check that `rendePrettyUnbound` never results in multiple consecutive spaces, tabs, or newlines?

Reply:
I added tests for `show-unbound` which inderectly tests `renderPrettyUnbounded`